rpm/libsailfishapp.spec

```diff
@@ -1,5 +1,5 @@
 Name: libsailfishapp
-Version: 0
+Version: 1
```

Comment:
I would prefer keeping zero here as better indication that the value is not accurate :)